README.md

```diff
@@ -19,12 +19,12 @@ Become independent from locked-in analytics systems and set up reliable tracking
 
 ## 🤓 Usage
 
-You can implement all sorts of front-end user events easily with walker.js. From e-commerce actions like product add to carts or order complete events to product and UX events like navigation, or filter usage, etc.
+You can implement all sorts of front-end user events easily with walker.js. From product and UX events like "newsletter signup", or filter usage, etc. to e-commerce actions like product add to carts or order complete events. It's up to you what you want to measure!
```

Comment:
I would cancel the last sentence, should be very clear from the ones before and only adds more text